src/main/java/com/denizenscript/depenizen/bukkit/objects/mythicmobs/MythicSpawnerTag.java

```diff
@@ -0,0 +1,341 @@
+
+    @Override
+    public String identify() {
+        return "mythicspawner@" + spawner.getName();
+    }
+
+    @Override
+    public String identifySimple() {
+        return identify();
+    }
+
+    @Override
+    public String toString() {
+        return identify();
+    }
+
+    public static void registerTag(String name, TagRunnable.ObjectInterface<MythicSpawnerTag> runnable, String... variants) {
+        tagProcessor.registerTag(name, runnable, variants);
+    }
+    public static ObjectTagProcessor<MythicSpawnerTag> tagProcessor = new ObjectTagProcessor<>();
+
+    public static void registerTags() {
+
+        // <--[tag]
+        // @attribute <MythicSpawnerTag.name>
+        // @returns ElementTag
+        // @plugin Depenizen, MythicMobs
+        // @description
+        // Returns the name of the MythicSpawner.
+        // -->
+        registerTag("name", (attribute, object) -> {
+            return new ElementTag(object.getSpawner().getName());
+        });
+
+        // <--[tag]
+        // @attribute <MythicSpawnerTag.cooldown>
+        // @returns DurationTag
+        // @plugin Depenizen, MythicMobs
+        // @mechanism MythicSpawnerTag.cooldown
+        // @description
+        // Returns an ElementTag(Number) of the MythicSpawner's cooldown.
+        // -->
+        registerTag("cooldown", (attribute, object) -> {
+            return new DurationTag(object.getSpawner().getCooldownSeconds());
+        });
+
+        // <--[tag]
+        // @attribute <MythicSpawnerTag.mythic_mobs>
+        // @returns ListTag(MythicMobsMobTag)
+        // @plugin Depenizen, MythicMobs
+        // @description
+        // Returns a ListTag of all active MythicMobs from this spawner.
+        // -->
+        registerTag("mythic_mobs", (attribute, object) -> {
+            ArrayList<MythicMobsMobTag> list = new ArrayList<>();
+            for (UUID uuid : object.getSpawner().getAssociatedMobs()){
+                list.add(MythicMobsMobTag.valueOf(uuid.toString()));
+            }
+            return new ListTag(list);
+        });
+
+        // <--[tag]
+        // @attribute <MythicSpawnerTag.mob_type>
+        // @returns ElementTag
+        // @plugin Depenizen, MythicMobs
+        // @mechanism MythicSpawnerTag.set_mob_type
+        // @description
+        // Returns an ElementTag for internal name the MythicMob mob type spawned.
+        // -->
+        registerTag("mob_type", (attribute, object) -> {
+            return new ElementTag(object.getSpawner().getTypeName());
+        });
+
+        // <--[tag]
+        // @attribute <MythicSpawnerTag.mobs>
+        // @returns ListTag(EntityTag)
+        // @plugin Depenizen, MythicMobs
+        // @description
+        // Returns a ListTag of all active MythicMobs from this spawner.
+        // -->
+        registerTag("mobs", (attribute, object) -> {
+            ArrayList<EntityTag> list = new ArrayList<>();
+            for (UUID uuid : object.getSpawner().getAssociatedMobs()){
```

Comment:
So many missing spaces. Make sure you have intelliJ configured to properly autoformat this stuff.